styles/map.css

```diff
@@ -0,0 +1,38 @@
+#map {
+  width: 100%;
+  height: 400px;
+  position: relative;
+  bottom: 260px;
+}
+
+.map__content {
+  display: flex;
+  justify-content: space-between;
+  width: 570px;
+  background-color: #ffffff;
+  position: relative;
+  top: 39px;
+  z-index: 1200;
+  padding: 70px 76px 54px;
+}
+
+.map__cooperation {
+  font-size: 20px;
+  line-height: 1.25;
+  text-align: left;
+  width: 154px;
+}
+
+.map__adress {
+  font-size: 16px;
+  font-family: "Arial";
```

Comment:
Add font family at the end: `sans-serif`

Reply:
fixed